### PR TITLE
Add BloodHound.app v1.1

### PR DESCRIPTION
**Pull request by ralphte**
_If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so._

After making all changes to the cask:
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:
- [x] Named the cask according to the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked that the cask was not already refused in [closed issues](https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed).

### CODE REVIEW

File: Casks/BloodHound.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'BloodHound' do
+  version '1.1'
+  sha256 '399ab1b17f68128cc5a2c143f3795ac74876fa48a117c284a09a7f5a196c6a8e'
+
+  url "https://github.com/adaptivethreat/BloodHound/releases/download/#{version}/BloodHound-darwin-x64.zip"
```
Comment (vitorgalvao) [inline Casks/BloodHound.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/BloodHound.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'BloodHound' do
```
Comment (vitorgalvao) [inline Casks/BloodHound.rb]: This needs to be all lowercase (same for the file name).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Please fill the template.

File: Casks/bloodhound.rb [hunk 3]
```diff
@@ -1,9 +1,11 @@
-cask 'BloodHound' do
+cask 'bloodhound' do
   version '1.1'
   sha256 '399ab1b17f68128cc5a2c143f3795ac74876fa48a117c284a09a7f5a196c6a8e'
 
   url "https://github.com/adaptivethreat/BloodHound/releases/download/#{version}/BloodHound-darwin-x64.zip"
-  name 'BloodHound'
+  appcast 'https://github.com/adaptivethreat/BloodHound/releases.atom',
+  		checkpoint: '09212808c04cbcb7d651570160c1140bc05c7791889ac1b49eab147e1c787eb3'
+  name 'bloodhound'
```
Comment (vitorgalvao) [inline Casks/bloodhound.rb]: This one was correct.

**Comment by vitorgalvao** [comment]
You still need to fill the template.

**Comment by ralphte** [comment]
Sorry I am not sure what you mean when you say fill the template?

**Comment by vitorgalvao** [comment]
Look at your first post.

**Comment by ralphte** [comment]
I see now. Thanks

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**